ldap/servers/plugins/memberof/memberof.c

```diff
@@ -421,6 +432,8 @@ memberof_postop_close(Slapi_PBlock *pb __attribute__((unused)))
     slapi_sdn_free(&_pluginDN);
     slapi_destroy_rwlock(config_rwlock);
     config_rwlock = NULL;
+    PR_DestroyLock(fixup_lock);
```

Comment:
It is initialized in init fct, I would prefer not to destroy it. Either init it in start (memberof_postop_start) and destroy it in close or simply init it in memberof_postop_init and do not bother to free it.

Reply:
I'll move it to the start function, that way dynamic plugins will work correctly and not break anything.